src/frontend/src/eth/utils/token.utils.ts

```diff
@@ -10,3 +11,6 @@ export const isSupportedErc20TwinTokenId = (tokenId: TokenId): boolean =>
 
 export const isNotSupportedErc20TwinTokenId = (tokenId: TokenId): boolean =>
 	!isSupportedErc20TwinTokenId(tokenId);
+
+export const tokenAddressToHex = (address: string): string =>
+	ethers.utils.hexZeroPad(address.toLowerCase(), 32);
```

Comment:
nth: I'm not familiar so as a nice to have, I would not be a comment about why `32`

Reply:
right, included comment